src/flownet/templates/webviz_ahm_config.yml.jinja2

```diff
@@ -32,7 +32,7 @@ pages:
         {%- for iteration in [iterations[0], iterations[-1]] %}
          - iter-{{ iteration }}
         {%- endfor %}
-        obsfile: ./observations.yamlobs
+        obsfile: ./observations_complete.yamlobs
```

Comment:
@anders-kiaer Is it possible to supply two observation file for the summary plotting? It would be nice to supply both the training and the test set and have them colored in a different way.

Reply:
Anders informed me that different colors, or multiple observation files, are not possible. However, you could add an extra yaml key called `comment` and write down test the observation belongs to train or test.

Like so:

```
    observations:
      - {value: 251, error: 10, date: 2003-01-01, comment: First measurement after sensor drift correction}
```

Reply:
So do you want one observation yaml file that has inside all data and a tag for each observation that indicates if corresponds to test or training?

Reply:
Yes - but also still the training and test files. So the goal of the PR doesn't change.